src/components/FullCarouselComponent/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import PropTypes from "prop-types"
+import "./style.sass"
+import {Container} from 'react-bootstrap'
+import AwesomeSlider from 'react-awesome-slider';
+import 'react-awesome-slider/dist/styles.css';
+
+export const FullCarouselComponent = ({data}) => {
+
+  return (
+
+    <Container>
+    <div className="carousel-div">
+     <AwesomeSlider>
+     {data.map(data => (
```

Comment:
Add a null check for if data prop in unavailable.

Reply:
Yes, I will. Thanks for reviewing it!